doc/container.md

```diff
@@ -312,9 +332,10 @@ By default, unlike other systems, persistent[^1] containers have no
 networking enabled.  All network access has to be set up explicitly.
 Currently two types of of container networks are supported:
 
- - `host`: one end of a VETH pair, or a physical Ethernet port
+ - `host`: one end of a VETH pair, or a physical Ethernet interface
```

Comment:
This could be something like a VLAN upper too, right? Maybe:

> - `host`: Delegate an interface managed by Infix -- e.g., one end of a VETH pair, or a physical interface -- to the container.

Reply:
Changed the wording a bit, wanted the bullet points to be brief.